example_scripts/generate_betagal_data_distributional.sh

```diff
@@ -0,0 +1,3 @@
+ #script that runs dataset generator with imaging parameters coming from distribution specified in the .cfg file
+python -m simSPI.linear_simulator.main simSPI/linear_simulator/configs/betagal_distribution.cfg
```

Comment:
Consider removing these bash scripts.

Reply:
you can just put what this bash script does in a readme, or in the documentation somehow